### ISSUE TITLE
The format of the experimental results "experiment.json" is not an valid json file

### ISSUE THREAD

**Issue by PurityFan** [Microsoft/nni#830]
**Short summary about the issue/question**:
The format of the experimental results "experiment.json" is not an valid json file.
Currently, the experiment result "experiment.json" give by NNI is like this:
`{
"id": "ZSYg8d6K", 
"params": {}, 
"searchSpace": {}, 
"maxTrialNum": 10000, 
"trialConcurrency": 5}[
{"id": "IYnpf", "status": "SUCCEEDED", "hyperParameters": {}, "finalMetricData": []"intermediate": []}
{"id": "IYnpf", "status": "SUCCEEDED", "hyperParameters": {}, "finalMetricData": []"intermediate": []}
{"id": "IYnpf", "status": "SUCCEEDED", "hyperParameters": {}, "finalMetricData": []"intermediate": []}
]`

In other words, NNI give the result as the format like:
{experiment_information}[trial_information1, trial_information2, trial_information3]

**But this is not a legal json file format.**

Need to make improvements in later versions.

**How to reproduce it**: 

Download "experiment.json" file in WebUI

**nni Environment**:
- nni version: 0.5.2
- nni mode(local|pai|remote): pai
- OS: Ubuntu
- python version: 3.5.2
- is conda or virtualenv used?: no
- is running in docker?: yes

**Comment by PurityFan** [comment]
problem solved by #899 . Thanks Lijiao~
